bindings/wasm/src/crypto/wasm_signature_options.rs

```diff
@@ -37,9 +37,9 @@ impl From<SignatureOptions> for WasmSignatureOptions {
   }
 }
 
-impl From<WasmSignatureOptions> for SignatureOptions {
-  fn from(options: WasmSignatureOptions) -> Self {
-    options.0
+impl From<&WasmSignatureOptions> for SignatureOptions {
+  fn from(options: &WasmSignatureOptions) -> Self {
+    options.clone().0
   }
 }
```

Comment:
Bit odd, maybe make two `From` implementations, one for the borrow and one for owned if you really need this?